internal/examples/proto/buf.gen.yaml

```diff
@@ -9,6 +9,11 @@ managed:
 
 plugins:
   - name: go
-    out: gen
+    out: gen/go
     opt: module=go.einride.tech/protobuf-bigquery/internal/examples/proto/gen
     path: ./build/protoc-gen-go
+
+  - name: json_schemas
```

Comment:
the `name` should typically be the exact name of the protoc plugin, i.e. `bq-json-schema` in this case